docs/tests/approvals-tests.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Approvals Tests
+---
+
+# Approvals Tests
+
+Approvals are based on the idea of the golden master. You take a snapshot of an
```

Comment:
I think a link to a something that explains the Golden Master will be very beneficial here. I don't think everyone will know what it is.

Reply:
I think I'm going to go ahead and remove that as it's not really necessary.